sdk/nodejs/.mise.toml

```diff
@@ -0,0 +1,10 @@
+[env]
+_.path = ["{{config_root}}/node_modules/.bin"]
+
+[hooks]
+postinstall = "yarn"
+
+[tools]
+"node" = "18"
```

Comment:
18.x, according to CI, is our current minimal version, hence why we're using it by default here. We have tests that try other combinations of versions, so it feels like we're going to catch more errors locally if we run the minimum version.

Reply:
In CI (in the merge queue) we run tests with the min & max supported versions, and I think there might also be a cronjob that runs the whole set of supported versions.

How would that fit into mise?

Reply:
From examples I've seen, it looks like the convention is to set everything up for the minimum version, and then in CI, loop over the versions and run `mise use` to override specific package versions for each iteration. I did look into whether we could make this explicit in Mise, but right now it looks (at least as far as I can tell) like we can't